common/check_fts_backlog

```diff
@@ -82,87 +74,84 @@ if __name__ == "__main__":
-                retvalue = OK
-                break
-            except Exception as error:
-                retvalue = CRITICAL
-                if result and result.status_code:
-                    errmsg = 'Error when trying to get info from %s : HTTP status code %s. [%s]' % (
-                        ftshost, str(result.status_code), str(error))
-                else:
-                    errmsg = 'Error when trying to get info from %s. %s' % (ftshost, str(error))
-        if retvalue == CRITICAL:
-            print("All attempts failed. %s" % errmsg)
-        WORST_RETVALUE = max(retvalue, WORST_RETVALUE)
-
-    if len(PROM_SERVERS):
-        for server in PROM_SERVERS:
-            try:
-                push_to_gateway(server.strip(), job='check_fts_backlog', registry=registry)
-            except:
-                continue
+
+                        if CHECK_BUSY and 'submitted' in channel and channel['submitted'] >= busylimit:
+                            url_activities = '%s/fts3/ftsmon/config/activities/%s?source_se=%s&dest_se=%s' % (ftshost, VO, src, dst)
+                            activities = {}
+                            try:
+                                s = requests.get(url_activities, verify=False, cert=(PROXY, PROXY))
+                                for key, val in s.json().items():
+                                    activities[key] = val['SUBMITTED']
+                            except Exception as error:
+                                pass
+                            busy_channels.append({'src': src, 'dst': dst, 'submitted': channel['submitted'],
+                                                  'activities': activities})
+                    summary = res['summary']
+                    hostname = hostname.replace('.', '_')
+                    # If printing these indiv is important, why not monitor them seperately? 
+                    print('%s : Submitted : %s' % (hostname, summary['submitted']))
+                    print('%s : Active : %s' % (hostname, summary['active']))
+                    print('%s : Staging : %s' % (hostname, summary['staging']))
+                    print('%s : Started : %s' % (hostname, summary['started']))
+
+                    if busy_channels != []:
+                        print('Busy channels (>%s submitted):' % busylimit)
+                        for bc in busy_channels:
+                            activities_str = ", ".join([("%s: %s" % (key, val)) for key, val in bc['activities'].items()])
+                            print('  %s to %s : %s submitted jobs (%s)' % (bc['src'], bc['dst'], bc['submitted'],
+                                                                           str(activities_str)))
+                            
+                    # Add to metrics
+                    backlog_count = summary['submitted'] + summary['active'] + summary['staging'] + summary['started']
+                    manager.gauge(
```

Comment:
This changes names too, but in a slightly more subtle way. However, I think the variable things should always come at the end of the statsd metric.

On a style note, I'd prefer combining lines 138 and 139. You could also split that long line (140) just before the first closing paren and it would look nice.